ansible/playbooks/service/postldap.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Manage Postfix service with Virtual Mail LDAP backend
+  hosts: [ 'debops_service_postldap' ]
+  become: True
+
+  environment: '{{ inventory__environment | d({})
+                   | combine(inventory__group_environment | d({}))
+                   | combine(inventory__host_environment  | d({})) }}'
+
+  roles:
+
+    - role: debops.etc_aliases/env
+      tags: [ 'role::etc_aliases', 'role::secret', 'role::postfix' ]
```

Comment:
Since the `debops.etc_aliases` role has been removed from the playbook, `debops.etc_aliases/env` subrole can also be removed.